config/environments/test.rb

```diff
@@ -12,8 +12,8 @@
   # preloads Rails for running tests, you may have to set it to true.
   config.eager_load = false
 
-  # Configure static asset server for tests with Cache-Control for performance.
-  config.serve_static_assets  = true
+  # Configure static file server for tests with Cache-Control for performance.
+  config.serve_static_files   = true
```

Comment:
Sorry... but are you sure this really changed from assets to files? I didn't see the rails 4.2 changelog yet.